### PR TITLE
Implemented emptyGrid function

### PR DESCRIPTION
**Pull request by Pravallika2812**

### CODE REVIEW

File: battleship.py [hunk 1]
```diff
@@ -62,7 +62,14 @@ def mousePressed(data, event, board):
 Returns: 2D list of ints
 '''
 def emptyGrid(rows, cols):
-    return
+    grid=[]
+    for i in range(rows):
+        l1 = []
```
Comment (Pragya28) [inline battleship.py:67]: Use relevant variable name instead of l1

**Review by Pragya28 — CHANGES_REQUESTED** [review]

File: battleship.py (hunk 1 above)
Comment (Pravallika2812) [reply battleship.py]: Modified Variable Name

File: battleship.py [hunk 2]
```diff
@@ -64,10 +64,10 @@ def mousePressed(data, event, board):
 def emptyGrid(rows, cols):
     grid=[]
     for i in range(rows):
-        l1 = []
+        cols = []
         for j in range(cols):
-            l1.append(EMPTY_UNCLICKED)
-        grid.append(l1)
+            cols.append(EMPTY_UNCLICKED)
+        grid.append(cols)
```
Comment (Pragya28) [inline battleship.py]: Did you test the function after updating @Pravallika2812?
Please check and make the required changes in case there is an error.

**Review by Pragya28 — CHANGES_REQUESTED** [review]

File: battleship.py (hunk 1 above)
Comment (Pravallika2812) [reply battleship.py]: Modified Variable name cols to col

**Review by Pragya28 — COMMENTED** [review]
Good work @Pravallika2812.

**Review by Pragya28 — APPROVED** [review]

**Pull request merged**